python/pyarrow/tests/test_fs.py

```diff
@@ -699,12 +699,12 @@ def test_get_file_info_with_selector(fs, pathfn):
 
         infos = fs.get_file_info(selector)
         if fs.type_name == "py::fsspec+s3":
-            # s3fs only lists directories if they are not empty, but depending
-            # on the s3fs/fsspec version combo, it includes the base_dir
+            # s3fs only lists directories if they are not empty
             # (https://github.com/dask/s3fs/issues/393)
```

Comment:
You can remove this issue link as well, because it is about the base dir issue (not the empty directories)